src/main/java/com/amazon/opendistroforelasticsearch/security/dlic/rest/api/InternalUsersApiAction.java

```diff
@@ -115,18 +115,23 @@ protected void handlePut(RestChannel channel, final RestRequest request, final C
         final ObjectNode contentAsNode = (ObjectNode) content;
         final SecurityJsonNode securityJsonNode = new SecurityJsonNode(contentAsNode);
 
-        // Don't allow user to add hidden, reserved or non-existent role
+        // Don't allow user to add hidden, reserved or non-existent rolesmapping
```

Comment:
more explicit comment can be  " Don't allow user to add non-existent role or a role for which ole-mapping is hidden or reserved "